RetailCoder.VBE/UI/RubberduckUI.fr.resx

```diff
@@ -1872,198 +1872,195 @@ End Sub</value>
   <data name="SettingsCaption_WindowSettings" xml:space="preserve">
     <value>Paramètres des fenêtres</value>
   </data>
-  <data name="CodeExplorer_OpenDesigner">
+  <data name="CodeExplorer_OpenDesigner" xml:space="preserve">
     <value>Ouvrir le designer</value>
   </data>
-  <data name="SourceControl_InitializeRepositoryTitle">
+  <data name="SourceControl_InitializeRepositoryTitle" xml:space="preserve">
     <value>Créer un nouveau repository local</value>
   </data>
-  <data name="RenameDialog_InterfaceRenameError">
+  <data name="RenameDialog_InterfaceRenameError" xml:space="preserve">
     <value>Impossible de renommer le membre '{0}' de l'interface.</value>
   </data>
-  <data name="Assert_NeitherParameterIsArray">
+  <data name="Assert_NeitherParameterIsArray" xml:space="preserve">
     <value>Ni [attendu] ni [réel] n'est un tableau.</value>
   </data>
-  <data name="Assert_UnexpectedArrayFormat">
+  <data name="Assert_UnexpectedArrayFormat" xml:space="preserve">
     <value>[attendu] et [réel] sont des tableaux. Considérez utiliser {0}.</value>
   </data>
-  <data name="Assert_ReferenceValueMismatch">
+  <data name="Assert_ReferenceValueMismatch" xml:space="preserve">
     <value>[attendu] est une référence, mais [réel] est une valeur.</value>
   </data>
-  <data name="Assert_GenericException">
+  <data name="Assert_GenericException" xml:space="preserve">
     <value>Une exception s'est produite lors de l'exécution des tests.</value>
   </data>
-  <data name="EmptyUIRefreshMessage">
+  <data name="EmptyUIRefreshMessage" xml:space="preserve">
     <value>pour analyser et traiter
 les projets dans le VBE.</value>
   </data>
-  <data name="CodeExplorer_ExportAll">
+  <data name="CodeExplorer_ExportAll" xml:space="preserve">
     <value>Exporter le projet...</value>
   </data>
-  <data name="Assert_UBoundMismatchFormat">
+  <data name="Assert_UBoundMismatchFormat" xml:space="preserve">
     <value>Dimension {0}: valeur UBound attendue {1}; valeur UBound réelle {2}. {3}</value>
   </data>
-  <data name="Assert_FailedMessageFormat">
+  <data name="Assert_FailedMessageFormat" xml:space="preserve">
     <value>Échec de l'assertion {0}. {1}</value>
   </data>
-  <data name="Assert_UnexpectedNullArraysFormat">
+  <data name="Assert_UnexpectedNullArraysFormat" xml:space="preserve">
     <value>[attendu] et [réel] sont 'Nothing'. Considérez utiliser {0}.</value>
   </data>
-  <data name="ToolsMenu_ExportProject">
+  <data name="ToolsMenu_ExportProject" xml:space="preserve">
     <value>Exporter le projet actif...</value>
   </data>
-  <data name="ImportCommand_OpenDialog_Filter_VBFiles">
+  <data name="ImportCommand_OpenDialog_Filter_VBFiles" xml:space="preserve">
     <value>Fichiers VB</value>
   </data>
-  <data name="GeneralSettings_EnableSourceControl">
-    <value>Activer le contrôle des source. Prendra effet au prochain démarrage.</value>
-  </data>
```

Comment:
Why was this dropped?

Reply:
@Vogel612 these resource keya are basically stale now that individual features can be toggled. Also there should be a dedicated settings tab for this, so the key would become `ExperimentalFeatureSettings_SourceControl` or something like it.